README.md

```diff
@@ -1,9 +1,11 @@
-# Installation
+Dotfiles, powered by [Homesick](https://github.com/technicalpickles/homesick).
```

Comment:
Download Source Code Pro: `https://github.com/adobe-fonts/source-code-pro/archive/latest/1.050R-it.zip`